components/table/demo/table.html

```diff
@@ -8,6 +8,9 @@
 			import '../../demo/demo-page.js';
 			import './table-test.js';
 		</script>
+		<script>
+			window.D2L = { LP: { Web: { UI: { Flags: { Flag: () => true } } } } };
```

Comment:
Why was this part needed? I would have expected any code checking a flag to use an optional chain on this. I guess if it's not defaulting to `true` after that?

Reply:
Copy pasted from another spot - I'll check if I can simplify

Reply:
Ah yup nevermind it wasn't defaulting to true so that was why this was needed.

Reply:
Cool. Wonder if it's worth a comment with the flag name just so we know we can remove this when either the flag defaults to true or is removed entirely? Otherwise this page could end up with some weird behaviour in the future!